images/kafka/Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN wget https://downloads.mesosphere.com/java/openjdk-jre-8u212b03-hotspot-linu
     mkdir /srv/jdk && \
     tar -xvvf /srv/openjdk-jre-8u212b03-hotspot-linux-x64.tar.gz --strip-components=1 -C /srv/jdk
 
+FROM openjdk:8-jdk as cruise-control
+RUN git clone --branch 2.0.77 https://github.com/linkedin/cruise-control.git && cd cruise-control/
```

Comment:
What is the purpose of the `cd`? `git` should exit non-zero if there is trouble, and I'm afraid the `cd` might confuse someone into believing *it* causes the next `RUN` to run in the subdir.

Reply:
good catch, removing this